### PR TITLE
Capi paid card styling fixes

### PR DESCRIPTION
**Pull request by Calanthe**
Small styling fixes for paid card cAPI component:
- The 'About' dropdown text is smaller now (everywhere also on the paid merchandising content and sticky band) 
- Article text link is underlined on hover state

screen shot 2016-02-08 at 12 55 57

### CODE REVIEW

File: static/src/stylesheets/module/commercial/paidfor/_paidfor-content.scss [hunk 1]
```diff
@@ -57,10 +57,12 @@
     }
 }
 
-.paidfor-box {
+.paidfor-box--header,
```
Comment (jamesgorrie) [inline static/src/stylesheets/module/commercial/paidfor/_paidfor-content.scss]: Is it not worth having this as `paidfor-box` and then adding the modifiers?
As far as I remember the BEM style would be to have `class="paidfor-box paidfor-box--header"` in the HTML?

Comment (uplne) [inline static/src/stylesheets/module/commercial/paidfor/_paidfor-content.scss]: In theory the whole thing is not using BEM. The correct way would be: .paidfor (main object) and then .paidfor__box (element of the object) and .paidfor__box--text (modification of the element).

Comment (uplne) [inline static/src/stylesheets/module/commercial/paidfor/_paidfor-content.scss]: And in totally ideal world we shouldn't even create new popup styles as we already have popup 
screen shot 2016-02-08 at 14 08 45

We should reuse that, add modifications (bg color) and some elements.

Comment (jamesgorrie) [inline static/src/stylesheets/module/commercial/paidfor/_paidfor-content.scss]: Having `.paidfor__box.paidfor__box--text` and `.paidfor-box.paidfor-box--text` are almost the same think I think. `.paidfor-box` could definitely be it's own block element.

I think my concern here is that having a modifier only (which is what these DOM elements have) moves away from BEM completely (a modifier should be used to modify and existing block).

In terms of the dropdown - that makes sense too... Is it abstracted enough to be reused?

Comment (Calanthe) [inline static/src/stylesheets/module/commercial/paidfor/_paidfor-content.scss]: Yeah, I wasn't sure about my approach. Thanks for suggestions, I will fix it.

**Comment by uplne** [comment]
:+1:

**Pull request merged**